### PR TITLE
AIX debug symbols support

### PR DESCRIPTION
**Pull request by andrew-m-leonard**
Signed-off-by: Andrew Leonard <anleonar@redhat.com>

### CODE REVIEW

File: hotspot/make/aix/makefiles/defs.make [hunk 1]
```diff
@@ -142,36 +142,28 @@ ifeq ($(JDK6_OR_EARLIER),0)
...
-
-        _JUNK_ := $(shell \
-          echo >&2 "INFO: STRIP_POLICY=$(STRIP_POLICY)")
-
-        ZIP_DEBUGINFO_FILES ?= 1
-
-        _JUNK_ := $(shell \
-          echo >&2 "INFO: ZIP_DEBUGINFO_FILES=$(ZIP_DEBUGINFO_FILES)")
-      endif
+      _JUNK_ := $(shell \
```
Comment (karianna) [inline hotspot/make/aix/makefiles/defs.make:145]: Does this seem to execute no matter what platform, is that right?

**Review by karianna — COMMENTED** [review]
LGTM, just the one nitpick

File: hotspot/make/aix/makefiles/defs.make (hunk 1 above)
Comment (andrew-m-leonard) [reply hotspot/make/aix/makefiles/defs.make:145]: this file is an AIX specific make file: hotspot/make/**aix**/makefiles/defs.make

**Comment by karianna** [comment]
@andrew-m-leonard - we've mirrored across over at Eclipse Adoptium.  Do you want to recreate this over there?

**Comment by andrew-m-leonard** [comment]
@karianna thanks will do

**Comment by karianna** [comment]
@andrew-m-leonard I assume we can close this as its done over at Adoptium now?

**Comment by andrew-m-leonard** [comment]
yes, thanks

**Pull request closed without merging**